README.md

```diff
@@ -15,40 +15,56 @@ Run `make test` to run all the tests.
 
 ## Redis Source
 
-The redis source has 2 modes
+The redis connector watches for new data being added in a *single* redis key supplied in `redis.key`. 
+Currently, the connector supports two type of Redis Data structures: `pubsub` & `stream`.
```

Comment:
Since we might have people new to Redis trying out this connector, let's write `Redis Data structures (DS)`, so that the later occurrences of DS are known to be this.